### PR TITLE
Enhancement of Login and Register UI, Enhancement of Navigation Bar and Making the Footer Dynamic.

### PR DESCRIPTION
**Pull request by RitabrataDas343**
Fixed Issue #24 and #43 for GSOC heat 5.0. Please review it.
Also Fixed Issue #27 in register forms.
Issue #48 also solved.

### CODE REVIEW

**Comment by RitabrataDas343** [comment]
NAVBAR



LOGIN


REGISTERATION

**Comment by kumanik** [comment]
The navbar seems very obnoxious try tonig the color down a bit

**Comment by RitabrataDas343** [comment]
Ok bhaiya .. I am toning the colour.

**Comment by RitabrataDas343** [comment]
Will it work, bhaiya???

File: base/templates/snippets/header.html [hunk 1]
```diff
@@ -1,189 +1,252 @@
...
+    }
+    
+    .search * {
+        outline: none;
+        box-sizing: border-box;
+    }
+    
+    .search__wrapper {
+        position: relative;
+        color: black;
```
Comment (RitabrataDas343) [inline base/templates/snippets/header.html:42]: Introduced black color to it

File: base/templates/snippets/header.html [hunk 2]
```diff
@@ -1,189 +1,252 @@
...
-
-
-
-
+    @media (min-width: 768px) {
+        html {
+            font-size: 16px;
+        }
+    }
+
```
Comment (RitabrataDas343) [inline base/templates/snippets/header.html:7]: Increased size

**Comment by RitabrataDas343** [comment]
FOOTER
ddddd

Will it work, Bhaiya??

**Pull request closed without merging**